charts/datadog/values.yaml

```diff
@@ -277,6 +277,9 @@ datadog:
     # datadog.systemProbe.collectDNSStats -- Enable DNS stat collection
     collectDNSStats: false
 
+    # datadog.systemProbe.enableConntrackAllNamespaces -- Enable conntrack updates from all network namespaces
+    enableConntrackAllNamespaces: true
```

Comment:
is `true` the default value in the `system-probe` configuration?

Reply:
if it wasn't, you have 2 options:
* change to `false`
* bump the minor version of the chart (`2.4.25` to `2.5.0`)